web/styles/portico/markdown.css

```diff
@@ -392,6 +396,24 @@
         }
     }
 
+    /* Style copy-to-clipboard button inside code blocks */
+    .copy-codeblock {
+        /* Invisible default button */
+        background: none;
+        border: none;
+        padding: 0;
+        position: absolute;
+        /* Position the button in the top-left corner */
```

Comment:
These comments twice mention top-left, when it's the top-right corner; I just fixed this before merging.